tests/specs/task/both_deno_json_selected/__test__.jsonc

```diff
@@ -0,0 +1,10 @@
+{
+  "args": "task other",
+  "cwd": "../../../testdata/task/both",
+  "output": "../../../testdata/task/both/deno_selected.out",
```

Comment:
Can you please move all of these files to be next to the relevant `__test__.jsonc` files. There should really be no relative file paths going up the tree in specs tests